<ui version="4.0" >
 <class>KDChartWidgetProperties</class>
 <widget class="QWidget" name="KDChartWidgetProperties" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Chart Properties</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="mTypeLabel" >
       <property name="text" >
        <string>Diagram Type</string>
       </property>
       <property name="buddy" >
        <cstring>mTypeCombo</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mTypeCombo" >
       <item>
        <property name="text" >
         <string>Bars</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Lines</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Pie</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Ring</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Polar</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="mSubTypeLabel" >
       <property name="text" >
        <string>Diagram Subtype</string>
       </property>
       <property name="buddy" >
        <cstring>mSubTypeCombo</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mSubTypeCombo" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>GroupBox</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="3" column="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1" colspan="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="3" >
       <widget class="QSpinBox" name="mGlobalLeadingBottomSB" />
      </item>
      <item row="2" column="3" >
       <widget class="QSpinBox" name="mGlobalLeadingRightSB" />
      </item>
      <item row="1" column="3" >
       <widget class="QSpinBox" name="mGlobalLeadingLeftSB" />
      </item>
      <item row="0" column="3" >
       <widget class="QSpinBox" name="mGlobalLeadingTopSB" />
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Global Leading Top</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Global Leading Bottom</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Global Leading Right</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Global Leading Left</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mCloseButton" >
       <property name="text" >
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
